EIPS/eip-1.md

```diff
@@ -67,54 +67,51 @@ These calls generally result in a "rough consensus" around what EIPs should be i
 
 *In short, your role as the champion is to write the EIP using the style and format described below, shepherd the discussions in the appropriate forums, and build community consensus around the idea.*
 
+
 ### EIP Process
 
 The following is the standardization process for all EIPs in all tracks:
 
 ![EIP Status Diagram](../assets/eip-1/EIP-process-update.jpg)
 
-**Idea** - An idea that is pre-draft. This is not tracked within the EIP Repository.
-
-**Draft** - The first formally tracked stage of an EIP in development. An EIP is merged by an EIP Editor into the EIP repository when properly formatted.
-
-**Review** - An EIP Author marks an EIP as ready for and requesting Peer Review.
-
-**Last Call** - This is the final review window for an EIP before moving to `Final`. An EIP editor will assign `Last Call` status and set a review end date (`last-call-deadline`), typically 14 days later.
-
-If this period results in necessary normative changes it will revert the EIP to `Review`.
-
-**Final** - This EIP represents the final standard. A Final EIP exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.
-
-A PR moving an EIP from Last Call to Final SHOULD contain no changes other than the status update. Any content or editorial proposed change SHOULD be separate from this status-updating PR and committed prior to it.
+- **Idea** - An idea that is pre-draft. This is not tracked within the EIP Repository.
+- **Draft** - The first formally tracked stage of an EIP in development. An EIP is merged by an EIP Editor into the EIP repository when it meets some initial requirements: It should be technically sound, the Title and Preamble must be correct, and all included Sections should be readable.  Problems with language like spelling, grammar, markup, etc. are only considered warnings.
+- **Review** - An EIP Author marks an EIP as ready for and requesting Peer Review.  An Editor will merge the EIP when it meets all EIP requirements: it should be a technically sound, complete specification, properly formatted, with correct language, markup and external references.
+- **Last Call** - This is the final review window for an EIP before moving to `Final`. An EIP editor will assign `Last Call` status and set a review end date (`last-call-deadline`), typically 14 days later.
+  - If this period results in necessary normative changes it will revert the EIP to `Review`.
+- **Final** - This EIP represents the final standard. A Final EIP exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.
+  - A PR moving an EIP from Last Call to Final SHOULD contain no changes other than the status update. Any content or editorial proposed change SHOULD be separate from this status-updating PR and committed prior to it.
```

Comment:
How about any status to any status?

Reply:
Not sure, but I don't think I changed this.

Reply:
I am ok that any changing status shall be a separate PR @Pandapip1

Reply:
Actually, this might conflict with enforcing EIPW only at status changes.

Reply:
I think there is value running EIPW on non-status PRs for the lines being changed.

Reply:
What has bitten me in the past is the rules changing in the course of drafting an EIP.  I'd much rather see warnings that I can plan to deal with later rather than hard errors that force me to deal with them right away.  Once it comes time to move to the next phase then indeed it's time to get everything up to the latest standards.

Reply:
huge +1 to "warning not error/merge-blocker"